src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,276 @@ redirects:
 
 <img
-  title="A screenshot showing where you can find the NRQL query builder button in the UI."
-  alt="A screenshot showing where you can find the NRQL query builder button in the UI."
-  src={accountsQueryBuilder}
+  title="Define your signal"
+  alt="A screenshot that shows you how to define your signal"
+  src={alertsDefineYourSignal}
 />
+  </Collapser>
+</CollapserGroup>
 
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: Use the query builder to start writing a NRQL query. Once you've written a valid query, you can create a condition from it.
-</figcaption>
+## Set your condition thresholds 
 
-The condition shows each host's CPU utilization. Use this NRQL query, or something similar that's relevant to what you're monitoring, to create a condition.
+If an alert condition is a container, then thresholds are the rules that each alert condition contains. As data streams into your system, the alert condition searches for any violations of these rules. If the alert condition sees data coming in from your system that has met all the conditions you've set, then it will create an incident. An incident is a signal that something is off in your system and you should take a look. 
 
-<img
-  title="A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname."
-  alt="A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname."
-  src={accountsNrqlQuery}
-/>
+Our team is creating an alert condition to look for any latency issues in our web transaction time. Now, we're going to create the rules this condition is going to look for. 
 
-<figcaption>
-  This query finds the average CPU utilization in an environment and then breaks it
-  down (facets) by individual host names.
-</figcaption>
+<CollapserGroup>
+  <Collapser
+    id="anomaly-threshold"
+    title="Anomaly threshold (recommended)"
+  >
 
-Once you've created a valid NRQL query, run it. Next, click **Create alert**.
+  Our team is creating this alert condition so that we're notified if web transaction time takes longer than usual. But let's say that we don't care how much longer web transaction time is, we just want to know if transaction time is behaving abnormally. For this specific use case, we'd recommend using our **anomaly threshold**. Our anomaly detection constantly evaluates your data to understand how your system normally behaves. By setting an anomaly threshold, you can use our anomaly detection to alert your team if web transaction time deviates from its expected performance. Since you only want to know if web transaction time is behaving unusually, you'd select **upper and lower** because you want notifications of _all_ deviations. But, if you only want alerts if web transaction times takes _longer_ than usual, you'd select **upper only.**
```

Comment:
```suggestion
Your team is creating this alert condition so you'll be notified if web transaction time takes longer than usual. But let's say you don't care how much longer web transaction time is, and you just want to know if transaction time is behaving abnormally. For this specific use case, we'd recommend using our **anomaly threshold**. Our anomaly detection constantly evaluates your data to understand how your system normally behaves. By setting an anomaly threshold, you can use our anomaly detection to alert your team if web transaction time deviates from its expected performance. Since you only want to know if web transaction time is behaving unusually, you'd select **upper and lower** because you want notifications of _all_ deviations. But, if you only want alerts if web transaction times takes _longer_ than usual, you'd select **upper only.**
```